### DISCUSSION TITLE
Problem when dragging image from same window

### DISCUSSION THREAD

**Discussion by themightyatom** [lllyasviel/Fooocus#3413] [Q&A]
I've been making an app to generate images to use as image prompts in Fooocus.
The issue is that the images import into Fooocus/Gradio, unless they come from another window (???)

https://youtu.be/mjNh8b4fYnk

I've made a video to illustrate the behaviour. I've tried with png, jpg, svg and dataURL. All work when dragging to another window, but nothing works from the same window. However, Fooocus itself manages this, being able to drag from generation window to inpaint etc.

Have been trying for 2 days, even round tripping to server and trying over https. 

What am I missing?

**Comment by mashb1t** [comment]
Works for me using the generated image in the same tab using Chrome.
What is the source of the image you're using?
Image and Fooocus have to be on the same physical computer in order for drag & drop to work.

**Reply by themightyatom** [reply]
I've used generated base64 images to start off with. When that didn't work I tried all other options I could think of.
The fact that they work from one browser window to another (also dataURLs) suggests they don't need to be on the same computer. I can also drag images directly from a google search. I've looked into everything as far as permissions, image tags etc. but can't get any further. There are no errors showing in the console. Should I post a minimalist example?

Thanks for taking the time to reply, and for such a completely awesome tool :)

**Reply by mashb1t** [reply]
Data URLs do worky but not paths to tempdir or outputs folder when drag & dropping down on a remote server. This has already been identified as the issue for this specific case.
TBH i'm not certain how to reproduce this. In general, images can be uploaded using the gradio file upload elements directly.

**Comment by themightyatom** [comment]
[image_example.zip](https://github.com/user-attachments/files/16441497/image_example.zip)
I dropped this in the javascript folder and added to ui_gradio_extensions.

<img width="611" alt="Screenshot 2024-07-31 140938" src="https://github.com/user-attachments/assets/cd0d1858-4853-42f9-b590-8a844213a927">
Here the image is loaded from a server.
